wire-library/wire-runtime-swift/src/main/proto/duration.proto

```diff
@@ -0,0 +1,116 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
This exists in the repo already [here](https://github.com/square/wire/blob/master/wire-library/wire-schema/src/jvmMain/resources/google/protobuf/duration.proto). It seems reasonable to point at that rather than duplicating if we can. (But would like to hear others' opinions too, @oldergod)

Reply:
Could we pull this out into a common folder?

Reply:
That seems reasonable too. I'm less familiar with Java file system organization conventions than all y'all, but seems like a reasonable approach. We will want a specific allow-list for iOS since we may not need all of them, but having the pool of .protos live somewhere common seems right.

Reply:
You should also be able to use the runtime jar as your input to the generation task and specify only the types you want. Wire can read inside jars.

Reply:
Do you have any examples of this?